
==================================================================================================
ReplicationVmFaultReasonForFault
==================================================================================================

.. describe:: ReplicationVmFaultReasonForFault

    

    
    .. py:data:: ReplicationVmFaultReasonForFault.invalidInstanceId

        The specified instanceId does not match the VirtualMachine instanceId

    
    .. py:data:: ReplicationVmFaultReasonForFault.invalidState

        VirtualMachine is in an invalid state

    
    .. py:data:: ReplicationVmFaultReasonForFault.notConfigured

        VirtualMachine is not configured for replication

    
    .. py:data:: ReplicationVmFaultReasonForFault.offlineReplicating

        VirtualMachine is in the process of creating an an offline instance.

    
    .. py:data:: ReplicationVmFaultReasonForFault.poweredOff

        VirtualMachine is powered off (and is not undergoing offline replication)

    
    .. py:data:: ReplicationVmFaultReasonForFault.poweredOn

        VirtualMachine is powered on

    
    .. py:data:: ReplicationVmFaultReasonForFault.suspended

        VirtualMachine is suspended (and is not undergoing offline replication)

    